content/docs/alerting/configuration.md

```diff
@@ -429,14 +429,14 @@ api_key: <secret>
 # A key used to map the alert to a team.
 [ routing_key: <string> ]
 
-# Describes the behavior of the alert (Critical, Acknowledgement, Info, Recovery).
-[ message_type: <string> ]
+# Describes the behavior of the alert (CRITICAL, WARNING, INFO, ACKNOWLEDGEMENT or RECOVERY).
+[ message_type: <tmpl_string> | default = 'CRITICAL' ]
```

Comment:
The default message type when the alert is firing is `CRITICAL`, if the alert is resolved then it is hardcoded to send `RECOVERY`.

There's no support for the `ACKNOWLEDGEMENT` message, as it's not in the white list and will be over-written to `CRITICAL` when the alert is firing, or be changed to `RECOVERY` when the alert is resolved.

We should either remove acknowledgement from this list and note that recovery is automatically sent, or add acknowledgement to the whitelist in alertmanager.

Reply:
Sure, let's remove ACKNOWLEDGEMENT from the list, since it is not used by alertmanager and there is a whitelist in the code.